test/functionalTest/cases/1158_v2_entity_already_exists/create_already_exists.test

```diff
@@ -96,14 +96,14 @@ Date: REGEX(.*)
 02. POST /v2/entities (Room1-Room)
 ==================================
 HTTP/1.1 422 Unprocessable Entity
-Content-Length: 69
+Content-Length: 56
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "description": "Entity already exists",
-    "error": "InvalidModification"
```

Comment:
This change in .test (and some other similar) corresponds to the following entry in CNR: "Fix: NGSIv2 "InvalidModification" errors changed to "Unprocessable" (according to rules defined in #1286)".

NTC (just a remark).